{:refedf: style="text-align: center;"}
![Image](picture.jpg)
{: refedf}

# Juli Jing, PhD
### Postdoctoral Associate on plant molecular biology and genome editing

## PROFESSIONAL SUMMARY
I am a highly-motivated molecular biologist specializing in cellular biology, genetics, genome editing and plant breeding. After completing a PhD and 3 years of postdoctoral research, I accumulated demonstrated success in technical proficiency such as molecular cloning, CRISPR/Cas9 genome editing, cytology, biochemistry and biotechnology. I also accumulated extensive knowledge about functional genome study, genome editing vector design, DNA homologous recombination and repair mechanism, enable me to develop more advanced genome editing methodologies, design and implement complex genome editing programs. During my 8 years of academic career, I have developed the specialist skillsets needed for scientific study, creativity, communicate complex concepts across domains, collaboration with multi-discipline team members, multi-task and organization abilities to work in a fast-paced, dynamic environment. I believe my solid background in genome editing, and strong work ethic will enable to adeptly manage complex research projects and collaborate effectively across functions.

## TECHNICAL SKILLS
**Molecular biology**
- DNA/RNA extraction
- PCR, RT-PCR, qPCR
- Molecular cloning, vector construction, recombinant DNA technology (Homologous alignment cloning, golden gate cloning, restriction enzyme cloning and Gibson-assembly)
- Genetic mapping: Map-based cloning, forward genetics
- _E.coli_ and _Agrobacterium_ transformation
- Plasmid extraction
- Yeast-two hybrid
- Split-luciferase complementary assay
- Subcellular localization
- Protein expression and purification in _E.coli._

**Genome editing**
- CRISPR/Cas9 target site gRNA design
- CRISPR/Cas9 vector construction
- _Agrobacterium_-mediated genetic transformation
- Maize and tomato tissue culture
- Screening of positive transgenic plants
- Mutation and phenotypic identification of transgenic positive plants

**Biochemistry (Protein assay)**
- Immunoassay
- Western blotting, SDS-PAGE gel electrophoresis
- Co-immunoprecipitation (Co-IP)
- Chromatin immunoprecipitation (ChIP)

**Cytogenetics and microscopy**
- Chromosome spreads, immunolocalization, fluorescence in situ hybridization (FISH)
- Microscopy: Stereomicroscopy, fluorescence and confocal microscopy, SIM and STED super-resolution microscopy
- Image processing software: Huygens, ImageJ, Imaris. Adobe Illustrator, Photoshop, GraphPad Prism, ect.

## RESEARCH EXPERIENCE
**_11/2023-present_** 

**Postdoctoral Associate - Cornell University** - Ithaca, NY, USA                                 

Supervisor: Wojtek Pawlowski
-	Contribute to book chapter in Cold Spring Harbor Protocol: Maize cytogenetics protocol
-	Identifying the function of maize meiotic protein by ChIP-seq.

**_10/2020-10/2023_** 

**Postdoctoral scientist - Max Planck Institute for Plant Breeding Research**- Cologne, Germany     

Supervisor: Raphael Mercier
-	Identified two novel Arabidopsis meiosis recombination regulators zyp1 HEI10oe and zyp1 recq4 which increase meiotic recombination rate by ~6 and ~10 times, has great application prospects in plant breeding. Applied one patent and published one Nature Communications paper based on it. 
-	Generated 6 Arabidopsis transgenic lines for proximity labeling to identify new meiotic crossover regulation proteins.
-	Proficient in immunofluorescence and microscopy techniques to study the protein localization on chromosome.
-	Won the top postdoctoral researcher fellowship of Germany.
-	Published 2 first author peer-reviewed publications. 
-	Communication: Hosted 3 years of department seminar and journal club, well-received presentations and posters are selected for 3 international conferences. Trained students and colleagues on molecular biology, cytology and biochemistry techniques.

**_08/2019-11/2019_** 

**Visiting Scholar- Cornell University** - Ithaca, NY, USA    

Supervisor: Wojtek Pawlowski
-	Conducted 3D immunolocalization with different antibodies in maize meiocytes.
-	Proficiently in using fluorescence and SIM super-resolution microscopy, ZEN and ImageJ image processing software.

**_09/2015-06/2020_**

**Ph.D candidate- China Agricultural University** - Beijing, China    

Supervisor: Yan He
-	Identified 5 new maize meiosis genes and elucidated their critical roles in meiosis process.  
-	Extensive experience in genome editing: knocked out 2 maize and 2 Arabidopsis genes by constructing and transferring dual customized gRNA CRISPR/Cas9 vectors to plants, leading to frameshift mutations, the corresponding mutant plants showed expected abnormal phenotypes. Proficient in plant genetic transformation and tissue culture.
-	Highly experienced in molecular cloning mythologies: constructed various vectors including 4 CRISPR/Cas9 genome editing vectors, 4 protein expression vectors, 40 yeast-two hybrid vectors, 2 subcellular localization vectors, 3 split luciferase complementation assay vectors. All these vectors expressed and functioned exceptionally well, produced expected results, and revealed the function of genes.
-	Deep understanding on DNA repair and recombination mechanism: published 5 research papers about the functional studies of DNA repair and recombination genes. Published 2 review papers about DNA recombination and repair mechanism. Reviewed several manuscript about DNA repair mechanism.
-	Leadership: Established and optimized the maize meiosis cytology and yeast-two hybrid experimental platform for the lab, wrote experimental tutorial for the lab. Trained and mentored 5 colleagues laboratory techniques. 
-	Published 8 peer-reviewed publications including 4 first-author publications.

## EDUCATION

**_09/2015-06/2020_**

**Ph.D.: Crop Genetics and Breeding**                                                        
China Agricultural University - National Maize Improvement Center - Beijing, China.
Dissertation: Map-based cloning and functional analysis of meiotic DNA double-strand break (DSB) repair factor ZmRAD51C and chromosome configuration regulator ICC1 in Maize. 

**_09/2011-06/2015_**

**Bachelor of Science: Seed Science and Engineering**     
Shandong Agricultural University - Taian, Shandong, China

## PEER REVIEWED PUBLICATIONS
1. Xu, W., Yu, Y., **Jing, J.**, Wu, Z., Zhang, X., You, C., Ma, H., Copenhaver, G.P., He, Y., and Wang, Y. SCFRMF mediates degradation of the meiosis-specific recombinase DMC1. Nature Communications (IF:14.919) 2023, 5044.
2. Jing, J.; Mercier, R., Meiosis: Deciphering the dialog between recombination and the synaptonemal complex. Current Biology (IF:10.834) 2022, 32, (21), R1235-R1237.
3. Durand, S*.; Lian, Q*.; **Jing, J***.; Ernst, M.; Grelon, M.; Zwicker, D.; Mercier, R., Joint control of meiotic crossover patterning by the synaptonemal complex and HEI10 dosage. Nature Communications (IF:14.919) 2022, 13, (1), 5999. (*co-first author).
4. **Jing, J.**; Wu, N.; Xu, W.; Wang, Y.; Pawlowski, W. P.; He, Y., An F-box protein ACOZ1 functions in crossover formation by ensuring proper chromosome compaction during maize meiosis. New phytologist (IF: 10.151) 2022, 235, (1), 157-172
5. Zhang, T.; **Jing, J.-L.**; Liu, L.; He, Y., ZmRAD17 Is Required for Accurate Double-Strand Break Repair During Maize Male Meiosis. Frontiers in plant science (IF: 5.753) 2021, 12.
6. **Jing, J.-L.**; Zhang, T.; Kao, Y.-H.; Huang, T.-H.; Wang, C.-J. R.; He, Y., ZmMTOPVIB enables DNA double-strand break formation and bipolar spindle assembly during maize meiosis. Plant Physiology (IF: 8.34) 2020, pp.00933.2020.
7. **Jing, J.**; Zhang, T.; Wang, Y.; Cui, Z.; He, Y., ZmRAD51C is Essential for Double-Strand Break Repair and Homologous Recombination in Maize Meiosis. International journal of molecular sciences (IF: 6.208). 2019, 20, (21).
8. **Jing, J. L.**; Zhang, T.; Wang, Y. Z.; He, Y., Advances Towards How Meiotic Recombination Is Initiated: A Comparative View and Perspectives for Plant Meiosis Research. International journal of molecular sciences (IF: 6.208). 2019, 20, (19).
9. Long, J. C.; Xia, A. A.; Liu, J. H.; **Jing, J. L.**; Wang, Y. Z.; Qi, C. Y.; He, Y., Decrease in DNA methylation 1 (DDM1) is required for the formation of (m) CHH islands in maize. Journal of Integrative Plant Biology (IF: 7.061). 2019, 61, (6), 749-764.
10. Wang, Y.; Jiang, L.; Zhang, T.; **Jing, J.**; He, Y., ZmCom1 Is Required for Both Mitotic and Meiotic Recombination in Maize. Frontiers in plant science (IF: 5.753). 2018, 9, 1005.

## PATENT
As the main inventor of the patent “Increasing meiotic recombination in plants by simultaneously inhibiting ZYP1 and RECQ4”, Ref AG1700 EP, EP23179262, Germany, June 2023

## FELLOWSHIP & SCHOLARSHIP
-	Alexander von Humboldt Research Fellowships for Postdoctoral Researchers, for the research proposal “Exploring the crossover decision mechanism within chromosome structure in Arabidopsis meiosis”, 2021-2023, (83,280€)
-	Postgraduate International Training and Promotion Scholarship, China Agricultural University, 2019, CNY ¥50,000
  
## AWARDS 
-	2020         Outstanding Graduate Award of Beijing

-	2020         Outstanding Graduate Award of China Agricultural University

-	2019         Monsanto Scholarship

-	2016-2018    Doctoral First-class Academic Scholarship

-	2015         Outstanding Graduate Award of Shandong Agricultural University

-	2014         Denghai Scholarship

## CONFERENCE PARTICIPATION
-	_06/2023_  **EMBO Meiosis Workshop**, Pamhagen, Austria

 	Poster: Unleashing meiotic crossovers by manipulating class I and class II crossover pathway
 	
-	_09/2022_  **Cologne-Versailles Symposium on Meiosis**, Cologne, Germany

 	Presentation: Simultaneously and massively increasing class I and class II crossovers in Arabidopsis thaliana
 	
-	08/2022_  **FASEB Science Research Conference: The Genetic Recombination and Genome Rearrangements Conference**, Steamboat Springs, CO, USA

 	Poster: Unleashing greater crossover potential in Arabidopsis thaliana

## INVITED PRESENTATIONS
-	_05/2021_ **Exploring the crossover decision mechanism within chromosome structure in Arabidopsis meiosis**. Max Planck Institute for Plant Breeding Research, Cologne, Germany
  
-	_02/2022_ **Studying meiotic recombination in zyp1 HEI10oe and zyp1 recq4**. Max Planck Institute for Plant Breeding Research, Cologne, Germany

-	_09/2022_ **Simultaneously and massively increasing class I and class II crossovers in Arabidopsis thaliana**. Cologne-Versailles Symposium on Meiosis, Cologne, Germany

-	_04/2023_ **Unleashing meiotic crossovers by manipulating class I and class II crossover pathway**. Max Planck Institute for Plant Breeding Research, Cologne, Germany

## PEER REVIEWER FOR SCIENTIFIC JOURNALS
Computational and Structural Biotechnology Journal. Plant Physiology. Nature Communications. Current Biology. Frontiers in Plant Science. Journal of Experimental Botany

## EDITORIAL ROLES
Review Editor for Plant Genetics, Epigenetics and Chromosome Biology and Plant Physiology section of Frontiers in Plant Science

## PROFESSIONAL CERTIFICATIONS
  Essentials of Research Administration, Cornell University, Credential ID 59535337
  
  Partnering with Technology Companies, Cornell University, Credential ID 59535338
  
  CRISPR Genome Editing: Rewriting DNA and the Future of Humanity, Cornell University, Credential ID 59535339
  
  How to Effectively Manage a Research Administration Team, Cornell University, Credential ID 59536340
  
  Best Practices for Global Research Partnerships: Benefits and Challenges, Cornell University, Credential ID 59535341
  

## LANGUAGES
 Chinese (Native proficiency), English (Professional working proficiency), German (Elementary proficiency)

## MEDIA REPORT PAGES
- <https://www.sciencedaily.com/releases/2022/10/221024131054.htm>
  
- <https://www.eurekalert.org/news-releases/968931>

- <https://phys.org/news/2022-10-evidence-theory-genetic-recombination.html>

- <http://www.postscientist.com/2022/10/evidence-for-new-theory-of-genetic.html>

- <https://scienmag.com/evidence-for-new-theory-of-genetic-recombination/>

- <https://www.youtube.com/watch?v=BPk3NRf0av8>

- <https://cab.cau.edu.cn/art/2020/6/19/art_23549_683866.html>





  
  
